driver-core/src/main/java/com/datastax/driver/core/schemabuilder/SchemaStatement.java

```diff
@@ -85,7 +78,7 @@ static void validateNotNull(Object value, String label) {
     }
 
     static void validateNotKeyWord(String label, String message) {
-        if (RESERVED_KEYWORDS.contains(label)) {
```

Comment:
This was broken, it should be `if (RESERVED_KEYWORDS.contains(label.toLowerCase()))`.